cmd/buildah/commit.go

```diff
@@ -26,6 +27,10 @@ var (
 			Name:  "format, f",
 			Usage: "`format` of the image manifest and metadata",
 		},
+		cli.StringFlag{
+			Name:  "reference-time",
+			Usage: "set the timestamp on the image to match the named `file`",
```

Comment:
The self-test for this checks that we get the same results from commit-then-push and the original push-while-committing paths, and after a while, the timestamps were the only difference in the results, so we needed some way to control them.  Using a reference file rather than offering to parse a date seemed more scripting-friendly (`touch` already does that, arguably better than we can), but it was a guess.